src/components/emotes.ts

```diff
@@ -0,0 +1,8 @@
+import { Emoji } from 'discord.js';
+import { CommandoClient } from 'discord.js-commando';
+
+export const emoteList: { [key: string]: Emoji } = {};
```

Comment:
You don't have to make this change, but for future reference, sometimes you would want to keep this variable private and maybe write a `getEmoteByName(name)` function that simply returns `emoteList[name]`. This simply hides the implementation detail of the data structure we decided to use to store the emotes. This pattern is similar to our `getDB` function, and it's something nice to think about what you write more complicated code later on.

Reply:
We don't need to make this change here because `emoteList` is simple enough